src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -153,15 +153,15 @@ public StorageConfiguration()
         [ConfigurationEntry("Storage.Encrypted.DisableBuffersPooling", ConfigurationEntryScope.ServerWideOnly)]
         public bool DisableEncryptionBuffersPooling { get; set; }
 
-        [Description("Max number of recyclable journals that will be reused. ")]
-        [DefaultValue(32)]
-        [MinValue(0)]
-        [ConfigurationEntry("Storage.MaxNumberOfRecyclableJournals", ConfigurationEntryScope.ServerWideOrPerDatabase)]
-        public int MaxNumberOfRecyclableJournals { get; set; }
 
         [Description("Disable further usage of sparse regions for the FS to reclaim free pages. In order to clear existing sparse regions you need to do that manually.")]
         [DefaultValue(false)]
         [ConfigurationEntry("Storage.DisableSparseRegions", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
         public bool DisableSparseRegions { get; set; }
+
+        [Description("EXPERT: Avoid shared journals between database & indexes.")]
+        [DefaultValue(false)]
+        [ConfigurationEntry("Storage.AvoidSharedJournals", ConfigurationEntryScope.ServerWideOrPerDatabase)]
+        public bool AvoidSharedJournals { get; set; }
```

Comment:
Storage.ShareJournals? Not sure...

Avoid for me suggests that in some situations it cannot be avoided?